doc/source/release/2.0.0-notes.rst

```diff
@@ -2,6 +2,82 @@
 NumPy 2.0.0 Release Notes
 =========================
 
+NumPy 2.0.0 is the first major release since 2006. It is the result of X months
+of development since the last feature release by Y contributors, and contains a
+large amount of exciting new features as well as a large amount of changes to
+both the Python and C APIs.
+
```

Comment:
Comment by @charris: start with a list of links to other docs that are relevant (migration guide, etc)